lib/widgets/recent_dm_conversations.dart

```diff
@@ -60,7 +66,28 @@ class _RecentDmConversationsPageBodyState extends State<RecentDmConversationsPag
             narrow: narrow,
             unreadCount: unreadsModel!.countInDmNarrow(narrow),
           );
-        }));
+        })),
+      floatingActionButton: Container(
+        constraints: const BoxConstraints(
+          minWidth: 137,
+          minHeight: 48),
```

Comment:
Where do these numbers come from?

When I run the PR, the button looks too big, with oversized padding:
![image](https://github.com/user-attachments/assets/88b0399a-f634-4db8-beb1-ff902dc17371)